pkg/model/components/cilium.go

```diff
@@ -168,5 +168,34 @@ func (b *CiliumOptionsBuilder) BuildOptions(o interface{}) error {
 		}
 	}
 
+	ingress := c.Ingress
+	if ingress != nil {
+		if ingress.Enabled == nil {
+			ingress.Enabled = fi.PtrTo(true)
+		}
+		if ingress.EnforceHttps == nil {
+			ingress.EnforceHttps = fi.PtrTo(true)
+		}
+		if ingress.EnableSecretsSync == nil {
+			ingress.EnableSecretsSync = fi.PtrTo(true)
+		}
+		if ingress.SecretsNamespace == "" {
+			ingress.SecretsNamespace = "cilium-secrets"
+		}
+		if ingress.LoadBalancerAnnotationPrefixes == "" {
+			ingress.LoadBalancerAnnotationPrefixes = "service.beta.kubernetes.io service.kubernetes.io cloud.google.com"
+		}
+		if ingress.DefaultLoadBalancerMode == "" {
+			ingress.DefaultLoadBalancerMode = "shared"
```

Comment:
Should shared really be the default? Cilium has dedicated as default.

Reply:
You're right makes more sense to use dedicated if cilium uses that as well